grpc_opa/options.go

```diff
@@ -64,3 +65,10 @@ func WithClaimsVerifier(claimsVerifier ClaimsVerifier) Option {
 		c.claimsVerifier = claimsVerifier
 	}
 }
+
+// WithEntitledServices overrides default EntitledServices
+func WithEntitledServices(entitledServices []string) Option {
```

Comment:
Variadic function fits in nicely and easier for sending end. 

func WithEntitledServices(entitledServices ...string) Option {

Reply:
Good idea, done.